config/routes.rb

```diff
@@ -7,11 +7,11 @@
 
   namespace :api, defaults: { format: :json } do
     get '/v1/metrics/', to: "metrics#index"
-    get '/v1/healthcheck', to: "healthcheck#index"
     get '/v1/organisations', to: 'organisations#index'
     get '/v1/document_types', to: 'document_types#index'
+    get '/v1/healthcheck', to: GovukHealthcheck.rack_response(
```

Comment:
For consistency with other apps I'd put this on the `/healthcheck` route. It's what most apps use:

https://github.com/alphagov/govuk-puppet/search?q=health_check_path

Reply:
Thank you @tijmenb, I have addressed your comment. I am not deleting `/v1/healthcheck` for the time being, because [I had to change Puppet to enable the new route](https://github.com/alphagov/govuk-puppet/pull/8619/files#diff-9ad5ee54919f692061c45771f662f350L112).

Once we have deployed puppet, and the new version is deployed, I will remove the endpoint `/v1/healthcheck`